_data/simple-icons.json

```
@@ -6957,6 +6957,16 @@
             "source": "https://medium.design/logos-and-brand-guidelines-f1a01a733592",
             "guidelines": "https://medium.design/logos-and-brand-guidelines-f1a01a733592"
         },
+        {
+            "title": "Medusa",
+            "hex": "8B5CF6",
+            "aliases": {
+                "aka": [
+                    "medusajs"
+                ]
+            },
+            "source": "https://medusajs.com"
```

```suggestion
            "source": "https://github.com/medusajs/medusa/blob/5b91a3503af41b94697a5c27c35268c29e9bd135/www/docs/static/img/logo.svg"
```

The website now only uses a PNG. Thankfully, this icon is available as an SVG through their GitHub!